### DISCUSSION TITLE
How do I install this program on Windows?

### DISCUSSION THREAD

**Discussion by Aprichnik** [idiap/coqui-ai-TTS#100] [Q&A] [answered]
how do I install this program on Windows so that there is such an interface?and so that you can run the program through a bat file as a stable diffusion 
![e47a6b1c17172de8cd763a87c298fc2d](https://github.com/user-attachments/assets/6b3dac3d-5534-4881-8aa5-7e4ed564d8e4)
https://huggingface.co/spaces/coqui/xtts

**Comment by eginhard** [comment, marked as answer]
If you click on _Files_ in the top right of the space, you will find the code: https://huggingface.co/spaces/coqui/xtts/tree/main You should be able to run it on Windows as well.
